proposed/2022/dotnet-nuget-config-spec.md

```diff
@@ -0,0 +1,369 @@
+# NuGet configuration CLI for dotnet.exe
+
+- Author: [Heng Liu](https://github.com/heng-liu)
+- GitHub Issue [8420](https://github.com/NuGet/Home/issues/8420)
+
+## Problem Background
+
+Currently, there is no NuGet configuration CLI for dotnet.exe. It's inconvenient for NuGet users to know about the NuGet configuration file locations and figure out where is the merged configuration coming from. 
+
+## Who are the customers
+
+This feature is for dotnet.exe users.
+
+## Goals
+Design and implement `dotnet nuget config` command.
+
+## Non-Goals
+Design and implement in the same way for NuGet.exe command.
+
+## Solution
+The following command will be implemented in the `dotnet.exe` CLI.
+
+### `dotnet nuget config`
+
+### Commands
+
+### Paths
```

Comment:
Sorry, I might be missing the context. Why don't default to this command? Why need `dotnet nuget config paths` instead of just `dotnet nuget config`?

Reply:
No worries! In our review meeting, we discussed and agreed that showing config file paths and showing config file contents should using different words. So we use `dotnet nuget config paths` for showing paths and `dotnet nuget config get` for showing contents.